phpunit.xml.dist

```diff
@@ -24,8 +24,6 @@
     <env name="SIMPLETEST_DB" value=""/>
     <!-- Example SIMPLETEST_SPARQL_DB value: sparql://127.0.0.1:8890/ -->
     <env name="SIMPLETEST_SPARQL_DB" value=""/>
-    <!-- Example SIMPLETEST_LEGACY_DB value: mysql://username:password@localhost/d6_databasename -->
-    <env name="SIMPLETEST_LEGACY_DB" value=""/>
```

Comment:
Deprecation test result is not shown, we need to comment out the following section below:

```
<env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```